tabconf/2021/2021-11-06-andrew-poelstra-miniscript.zh.md

```diff
@@ -1,21 +1,18 @@
 ---
-title: Andrew Poelstra - Miniscript (2021-11-06)
+title: Miniscript 
 transcript_by: Michael Folkson
 translation_by: Ajian
 categories: ['conference']
 tags: ['miniscript', 'wallet']
+speakers: ['Andrew Poelstra']
+date: 2021-11-06
+media: https://www.youtube.com/watch?v=mVihRFrbsbc&t=6470s
```

Comment:
Thank you for going back and fixing this to follow the new guidelines. 
Maybe you can also do the same in a separate PR for transcripts in TABConf 2022 and Advancing Bitcoin

Reply:
Thanks very much. Sure, I will make a new PR to let them following the new guidelines.

I think it will be convenient to accept this PR at first. Then I can sync this better version and start my new work.

Reply:
sounds good!